source/egg/core/Heap.hh

```diff
@@ -91,6 +91,8 @@ protected:
 
 } // namespace EGG
 
+#ifdef KINOKO_EGG_OPERATOR_NEW
+#pragma GCC visibility push(hidden)
```

Comment:
I thought based off the bug you linked, GCC won't actually hide these symbols. Does that mean this pragma does nothing? I think we should just get rid of it then.

Reply:
I still left these (along with some of the cmake changes) as an intent of we actually want (although I'm not optimistic of this being fixed in gcc soon). I don't mind removing them though